Neos.ContentRepository.Core/Classes/Feature/ContentStreamCommandHandler.php

```diff
@@ -130,14 +129,13 @@ private function handleReopenContentStream(
      */
     private function handleForkContentStream(
         ForkContentStream $command,
-        ContentRepository $contentRepository
+        CommandHandlingDependencies $commandHandlingDependencies
     ): EventsToPublish {
-        $this->requireContentStreamToExist($command->sourceContentStreamId, $contentRepository);
-        $this->requireContentStreamToNotBeClosed($command->sourceContentStreamId, $contentRepository);
-        $this->requireContentStreamToNotExistYet($command->newContentStreamId, $contentRepository);
+        $this->requireContentStreamToExist($command->sourceContentStreamId, $commandHandlingDependencies);
+        $this->requireContentStreamToNotBeClosed($command->sourceContentStreamId, $commandHandlingDependencies);
 
-        $sourceContentStreamVersion = $contentRepository->getContentStreamFinder()
-            ->findVersionForContentStream($command->sourceContentStreamId);
+        // TODO: This is not great
```

Comment:
```suggestion
```

Huuuch a todo i did no know about?
If its about the `getContentStreamFinder` thing that we should better not use i have a followup planned ^^ but else what is this about?